src/app/pages/rpc-send-transfer/rpc-send-transfer.context.ts

```diff
@@ -5,15 +5,19 @@ import type { UtxoResponseItem } from '@leather.io/query';
 
 import type { TransferRecipient } from '@shared/models/form.model';
 
-import type { RpcRequestContext } from '@app/common/rpc/use-rpc-request';
+interface RpcRequestContext {
```

Comment:
Temp relocation until I refactor to share new `RpcRequestContext` from the feature.